corehq/apps/callcenter/management/commands/rebuild_call_center_datasources.py

```diff
@@ -13,7 +13,6 @@
 
 
 class Command(BaseCommand):
-    args = '<domain domain ...>'
```

Comment:
this one has the same issue where the list of domains will be in options['domains']